docs/extensions/extensions.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 9
+---
+
+# 🔌 Extensions
+
+This guide introduces extensions for Scaffold-ETH 2 and explains how to use and create extensions.
+
+## What are Extensions?
+
+Extensions are modular add-ons for Scaffold-ETH 2 that provide additional functionality or serve as examples for specific features. They offer several benefits:
+
+- Seamless integration with the base Scaffold-ETH 2 project
+- Quick addition of new features, pages, contracts, or components
+- Compatibility with Scaffold-ETH 2 core updates and improvements
+
+Extensions are compact packages containing specific code (such as a smart contract or UI component) that automatically integrate with the latest version of Scaffold-ETH 2 when initializing a new project via npx. They are starting points for your project, not finished products.
+
+## Using Extensions
+
+To use an extension when creating a new Scaffold-ETH 2 project, run:
+
+```bash
+npx create-eth@latest -e {your-github-userName}/{extension-repo-name}:{branch-name}
+```
```

Comment:
Nitpicks, can ignore if you think they are too obvious!

- Would be nice to add an example? Like we do in: https://github.com/scaffold-eth/se-2-docs/blob/dbda695635bf52761a5409dbea71a83d4d570e8a/docs/extensions/availableExtensions.md?plain=1#L15
- "your" in `{your-github-userName}` could lead noob developers to think they can only install their own extensions? Just talking out loud a initial thought that came when I saw it.

Reply:
> I think it looks good, maybe when we have another example from one us with a githubusername/extension it can also be added there, so that it won't be confusing. (since its only as subgraph -- and no gh username in the example)

Yeah! Maybe we could use the youtube tutorial repo `https://github.com/ChangoMan/charts-extension` if we wanted to have the `{github-username}/{extension-repo-name}` format right now

Reply:
ah this is great - just changed to hunters repo!